src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -191,20 +231,131 @@ public CaseDetails update(String authorisation, CCDCase ccdCase, CaseEvent caseE
                 .ignoreWarning(true)
                 .build();
 
-            return updateCoreCaseDataService
-                .update(authorisation, eventRequestData, ccdCase, ccdCase.getId());
+            StartEventResponse startEventResponse = startUpdate(authorisation, eventRequestData, caseId);
+
+            CaseDataContent caseDataContent = CaseDataContent.builder()
+                .eventToken(startEventResponse.getToken())
+                .event(Event.builder()
+                    .id(startEventResponse.getEventId())
+                    .summary("CMC case update")
```

Comment:
not really convinced this is needed. and if it is it should probably come from the event not be hardcoded in update where its really giving no value